### PR TITLE
DOC add link to the hub with skops filter

### LINKED ISSUE #127: Add a link to the hub with all `skops` models filtered
xref: https://github.com/huggingface/moon-landing/issues/3757#issuecomment-1236708571

> I'm also not sure if we should add `skops` as a separate library. We probably don't want to put too many libraries in that list, since it will diminish its usability.
> 
> Something we could do, is to add a link with the filter for `skops` on the `skops`'s documentation so that users can simply click and see all repos. Would that be a good alternative?

We could also potentially add a link to model card with the same link: https://github.com/huggingface/moon-landing/issues/3757#issuecomment-1236744472

> Add something somewhere in the modelcard that says that the repo was automatically created with skops and a link to skops.

### PR DESCRIPTION
**Pull request by adrinjalali**
Fixes #127 

cc @osanseviero @skops-dev/maintainers

### CODE REVIEW

**Review by BenjaminBossan — APPROVED** [review]
LGTM

We should probably address #114 soon to avoid showing so many CI-related repos.

**Comment by adrinjalali** [comment]
CI finally green

File: docs/hf_hub.rst [hunk 1]
```diff
@@ -13,6 +13,8 @@ things, gives you the following benefits:
 - Collaborating with others on a model through discussions and pull requests
 - Convenient sharing of models with the community
 
+You can see all the models uploaded to the Hugging Face Hub using this library
+`here <https://huggingface.co/models?other=skops>`_`
```
Comment (BenjaminBossan) [inline docs/hf_hub.rst:17]:
```suggestion
`here <https://huggingface.co/models?other=skops>`_.
```

**Review by BenjaminBossan — CHANGES_REQUESTED** [review]
Some links are broken. Maybe you can check if your editor has an RST mode with a command to add hyperlinks, since it's so easy to mess them up.

**Comment by adrinjalali** [comment]
hehe, sometimes my editor puts two backticks when I type one, annoying.

**Comment by BenjaminBossan** [comment]
@adrinjalali From my POV this can be merged but it needs to be updated.

**Comment by adrinjalali** [comment]
@BenjaminBossan you need to change your "review" here for me to have the green button 😁

**Review by BenjaminBossan — APPROVED** [review]

**Pull request merged**